imaginate_api/app.py

```diff
@@ -184,6 +190,43 @@ def delete_image(id):
   fs.delete(res._id)
   return info
 
+#Image Verification Routes
+
+# GET /image/verification-portal: returns HTML page for image verification
+@app.route("/image/verification-portal", methods=["GET"])
+def verification_portal():
+    obj = db['fs.files'].find_one({
+      '$or': [
+        { 'status':'unverified' },
+        { 'status': { '$exists': False } }
+      ]
+    })
+    if obj:
+        grid_out = fs.find_one({ "_id": obj['_id'] })
+        data = grid_out.read()
+        base64_data = b64encode(data).decode('ascii');
+        return render_template('verification_portal.html', id=obj['_id'], img_found=True, img_src=base64_data)
+    return render_template('verification_portal.html', img_found=False)
+
+# POST /image/update-status: used to mark images as verified / rejected
+@app.route("/image/update-status", methods=["POST"])
+def update_status():
+    status = request.form['status']
+    if status:
+        query_filter = { '_id': ObjectId(request.form['id']) }
+        update_operation = { "$set" : { "status" : status } }
```

Comment:
We could use the ImageStatus class to verify that this is a valid status, it shouldn't matter since it'll be hardcoded in frontend but will be good if we add/modify statuses in future